identity/client/src/utility/api/users/index.ts

```diff
@@ -1,10 +1,11 @@
-import { ApiDefinition, apiDelete, apiGet, apiPost, apiPut } from "../request";
+import { ApiDefinition, apiDelete, apiPost, apiPut } from "../request";
 import { SearchResponse } from "src/utility/api";
+import { EntityData } from "src/components/entityList/EntityList";
 
 export const USERS_ENDPOINT = "/users";
 
-export type User = {
-  id: string;
+export type User = EntityData & {
+  key: number;
```

Comment:
Also I don't fully understand the difference between id, key and username. Maybe you can elaborate a bit more on this.

Reply:
username is the name chosen by user and we use it for login and also in search api. id and key can be used interchangeably for now (only their type is different) but may be later (when implementation of all entities is done) we can revise and make them one.